common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/ResponseType.java

```diff
@@ -0,0 +1,33 @@
+package com.microsoft.identity.common.internal.providers.oauth2;
+
+/**
+ * The response type values defined by the
+ * [OAuth 2.0](https://tools.ietf.org/html/rfc6749) and
+ * [OpenID Connect Core 1.0](http://openid.net/specs/openid-connect-core-1_0.html)
+ **/
+
+public final class ResponseType {

```

Comment:
This is fine for now.  Let's discuss where to put these constants... I took a somewhat different approach.